### PR TITLE
Better matching to video

### PR DESCRIPTION
**Pull request by itzikg**
Better matching to video

### CODE REVIEW

File: resources/lib/SubtitleHelper.py [hunk 1]
```diff
@@ -14,19 +18,71 @@ def log(module, msg):
...
                                                 int(item['season']),
                                                 int(item['episode']),)
-                                              ).replace(" ","+")      
+                                              )#.replace(" ","+")      
     else:
         if str(item['year']) == "":
           item['title'], item['year'] = xbmc.getCleanMovieTitle( item['title'] )
     
-        search_string = item['title'].replace(" ","+")
+        search_string = item['title']#.replace(" ","+")
```
Comment (slmosl) [inline resources/lib/SubtitleHelper.py]: did you leave this comment intentionally?

**Comment by slmosl** [comment]
@itzikg interesting code - did you check it on various inputs?

File: resources/lib/SubtitleHelper.py (hunk 1 above)
Comment (itzikg) [inline resources/lib/SubtitleHelper.py]: Yes it intentionally

**Comment by itzikg** [comment]
I checked it with different video files from various sources.

**Pull request closed without merging**